src/templates.js

```diff
@@ -19,6 +19,11 @@ import { itemsMap } from './data/maps'
  */
 export const CROW_ATTACKED = (_, crop) => `Oh no a crow ate your ${crop.name}!`
 
+export const CROWS_DESTROYED = (_, numCropsDestroyed) =>
```

Comment:
nit: This template needs JSDoc annotations to be consistent with the other ones.